### ISSUE TITLE
java.util.regex.PatternSyntaxException while trying to capture a screenshot

### ISSUE THREAD

**Issue by constantine-ivanov** [onesky/auto-screenshot-android#11]
Stacktrace:
```
01-21 15:45:44.630 15483-15483/com.truecaller.debug E/AndroidRuntime: FATAL EXCEPTION: main
                                                                      Process: com.truecaller.debug, PID: 15483
                                                                      java.lang.RuntimeException: Error receiving broadcast Intent { act=com.oneskyapp.screenshot.ScreenshotButtonService.CAPTURE_ACTION flg=0x10 } in com.oneskyapp.screenshot.OneSkyScreenshotHelper$1@149ec766
                                                                          at android.app.LoadedApk$ReceiverDispatcher$Args.run(LoadedApk.java:933)
                                                                          at android.os.Handler.handleCallback(Handler.java:739)
                                                                          at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                          at android.os.Looper.loop(Looper.java:145)
                                                                          at android.app.ActivityThread.main(ActivityThread.java:5832)
                                                                          at java.lang.reflect.Method.invoke(Native Method)
                                                                          at java.lang.reflect.Method.invoke(Method.java:372)
                                                                          at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1399)
                                                                          at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1194)
                                                                       Caused by: java.util.regex.PatternSyntaxException: Syntax error in regexp pattern near index 1:
                                                                      +(.*)? more
                                                                       ^
                                                                          at java.util.regex.Pattern.compileImpl(Native Method)
                                                                          at java.util.regex.Pattern.compile(Pattern.java:411)
                                                                          at java.util.regex.Pattern.<init>(Pattern.java:394)
                                                                          at java.util.regex.Pattern.matches(Pattern.java:424)
                                                                          at java.lang.String.matches(String.java:1581)
                                                                          at com.oneskyapp.screenshot.utils.StringUtils.compareStrings(StringUtils.java:264)
                                                                          at com.oneskyapp.screenshot.OneSkyScreenshotHelper.captureLocalizableStrings(OneSkyScreenshotHelper.java:364)
                                                                          at com.oneskyapp.screenshot.OneSkyScreenshotHelper.captureLocalizableStrings(OneSkyScreenshotHelper.java:345)
                                                                          at com.oneskyapp.screenshot.OneSkyScreenshotHelper.captureLocalizableStrings(OneSkyScreenshotHelper.java:345)
                                                                          at com.oneskyapp.screenshot.OneSkyScreenshotHelper.captureLocalizableStrings(OneSkyScreenshotHelper.java:345)
                                                                          at com.oneskyapp.screenshot.OneSkyScreenshotHelper.captureLocalizableStrings(OneSkyScreenshotHelper.java:345)
                                                                          at com.oneskyapp.screenshot.OneSkyScreenshotHelper.captureLocalizableStrings(OneSkyScreenshotHelper.java:345)
                                                                          at com.oneskyapp.screenshot.OneSkyScreenshotHelper.captureLocalizableStrings(OneSkyScreenshotHelper.java:345)
                                                                          at com.oneskyapp.screenshot.OneSkyScreenshotHelper.captureLocalizableStrings(OneSkyScreenshotHelper.java:345)
                                                                          at com.oneskyapp.screenshot.OneSkyScreenshotHelper.captureLocalizableStrings(OneSkyScreenshotHelper.java:345)
                                                                          at com.oneskyapp.screenshot.OneSkyScreenshotHelper.captureLocalizableStrings(OneSkyScreenshotHelper.java:345)
                                                                          at com.oneskyapp.screenshot.OneSkyScreenshotHelper.getScreenshot(OneSkyScreenshotHelper.java:255)
                                                                          at com.oneskyapp.screenshot.OneSkyScreenshotHelper.capture(OneSkyScreenshotHelper.java:231)
                                                                          at com.oneskyapp.screenshot.OneSkyScreenshotHelper$1.onReceive(OneSkyScreenshotHelper.java:116)
                                                                          at android.app.LoadedApk$ReceiverDispatcher$Args.run(LoadedApk.java:923)
                                                                          	... 8 more
```

Happens every time I press the capture button.

**Comment by constantine-ivanov** [comment]
It would be nice to encapsulate text from a string resource into \Q\E so symbols like "+" don't produce invalid expressions.

**Comment by bretdabaker** [comment]
Fixed in [v0.7.3](https://github.com/onesky/auto-screenshot-android/releases/tag/v0.7.3)

Please try and let me know if the problem persists.
